requirements/base.txt

```diff
@@ -1,195 +1,224 @@
 #
-# This file is autogenerated by pip-compile
+# This file is autogenerated by pip-compile with python 3.8
 # To update, run:
 #
 #    ./bin/compile_dependencies.sh
 #
 alabaster==0.7.12
     # via sphinx
-amqp==2.5.2
+amqp==5.0.6
     # via kombu
-babel==2.8.0
-    # via
-    #   flower
-    #   sphinx
-billiard==3.6.1.0
+babel==2.9.1
+    # via sphinx
+billiard==3.6.4.0
     # via celery
-celery-once==3.0.1
-    # via -r requirements/base.in
-celery==4.4.0
+bitly-api-py3==0.3
+    # via django-docutils
+celery==5.1.2
```

Comment:
Celery 5 is not supported with Flower, we should stick to Celery 4 for the time being